romancal/datamodels/container.py

```diff
@@ -486,3 +528,27 @@ def get_crds_parameters(self):
             crds_header |= model.get_crds_parameters()
 
         return crds_header
+
+
+def make_file_with_index(file_path, idx):
```

Comment:
As discovered by @mairanteodoro this may need to be fixed but I don't think it should hold the PR. Will file another ticket.